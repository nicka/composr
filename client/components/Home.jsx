import React from 'react';
import { Link } from 'react-router-dom';

const Home = () =>
  <div>
    <h1>We{'\''}ve been acquired by BandLab!</h1>
    <p>
      <a
        href="https://blog.bandlab.com/bandlab-welcomes-composr"
        target="_blank"
        rel="noopener noreferrer"
      >Read the announcement</a> â€“ July 7, 2016
    </p>
    <p>
      <Link replace to="/faq">FAQ</Link>
    </p>
  </div>;

export default Home;
