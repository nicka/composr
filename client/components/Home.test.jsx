import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Home from './Home';

describe('Home', () => {
  it('renders the component', () => {
    const component = renderer.create(
      <MemoryRouter>
        <Home />
      </MemoryRouter>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
