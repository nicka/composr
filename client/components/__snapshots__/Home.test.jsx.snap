// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the component 1`] = `
<div>
  <h1>
    We
    '
    ve been acquired by BandLab!
  </h1>
  <p>
    <a
      href="https://blog.bandlab.com/bandlab-welcomes-composr"
      rel="noopener noreferrer"
      target="_blank"
    >
      Read the announcement
    </a>
     â€“ July 7, 2016
  </p>
  <p>
    <a
      href="/faq"
      onClick={[Function]}
    >
      FAQ
    </a>
  </p>
</div>
`;
